engine/gui/src/gui.cpp

```diff
@@ -1736,6 +1736,21 @@ Result DeleteDynamicTexture(HScene scene, const dmhash_t texture_hash)
                         lua_settable(L, -3);
                     }
 
+                    if (ia->m_GamepadConnected) 
+                    {
+                        lua_pushliteral(L, "gamepad_name");
+                        if (ia->m_TextCount == 0)
```

Comment:
Same as above, you can use `lua_pushlstring` for both cases.

Reply:
Fixed in https://github.com/defold/defold/pull/4970/commits/5a5b367269321c9773c1b9e96d219773ce07b9fe